uint8[] idArray
uint8 idCnt
---
float64[] getData
